### DISCUSSION TITLE
Email notification on "task due" date

### DISCUSSION THREAD

**Discussion by registry** [Leantime/leantime#835] [Ideas]
Im my "Leantime" usage pattern, i would like to assign to some tasks a "reminder", a email notification of me(or creater)  when "due" date is approaching and task is still not in "Done" state.

Just as an idea....

**Comment by marcelfolaron** [comment]
Totally agree. What reminder timeframes would you like?
Three days before? Day before? On the date?

**Reply by registry** [reply]
Well, i  my particular case  a reasonable default of "1 day before" is totally ok, as my tasks are "fast to acompilsh, one needs only to be notified to do it"

But i general case it might be different from project to project.

I would go for something  semi-flexible, by offering something like 
"no reminder"
"1h before" (dunno if that is valuablu)
"1 day before" (good default)
"3 days before"
"week before"
"choosen date for reminder"
with default of "1 day before"...
